File: Programas em Phyton/Ex35.py
numeros = [20]
constante = 5
for i in range (0, 20, 1):
    x = int(input('Digite 20 números: '))
    numeros.append(x)
print('Os números digitados multiplicados por 5 são: \n')
for i in range (1, 21, 1):
    numeros[i]=constante*numeros[i]
    print(f'{numeros[i]}')

File: Programas em C/Ex35.c
#include<stdio.h>
#include<stdlib.h>
#include<ctype.h>
int main(int argc, char const *argv[])
{
    int numeros[20];
    int constante;
    constante=5;

    printf("Digite os 20 numeros: ");
    for(int i=0; i<20; i++)
    {
        scanf("%i", &numeros[i]);
    }
    printf("Os numeros digitados multiplicados por 5 sao: \n");
    for(int i=0; i<20; i++)
    {
        if(numeros[i] = constante * numeros[i]);
        printf("%i\n", numeros[i]);
    }

    return 0;
}
